lib/components/artist/related_artists/related_artist.js

```diff
@@ -49,7 +49,18 @@ const styles = StyleSheet.create({
 });
 
 export default Relay.createContainer(RelatedArtist, {
-  initialVariables: { width: 150, height: 100 },
+  initialVariables: { imageWidth: 150, imageHeight: 100 },
+
+  prepareVariables: prevVariables => {
+    const {width, height} = Dimensions.get('window');
+    let w = width;
```

Comment:
@alloy I put a breakpoint here and it's getting called before `render()` but for some reason `render()` uses the `initialVariables` instead of these modified ones. Is there something else I need to do to make sure `prepareVariables` overrides `initialVariables`?